seqr/views/apis/report_api.py

```diff
@@ -111,7 +115,7 @@ def _get_sample_counts(sample_q, data_type_key='dataset_type'):
 ]
 
 
-@analyst_required
+@airtable_enabled_analyst_required
```

Comment:
The 2 endpoints with the added airtable check in this file are already not available in the UI for local users so they would not be hitting these endpoints anyways, but its better to be explicit here